info/flow_and_pressure.md

```diff
@@ -0,0 +1,42 @@
+
+## Levers
+
+In a lever machine, a piston is either forced down by a spring or the arm of a human, which then forces water through the puck. Levers are unique because they can provide nearly unlimited water debit. To the system. If you wanted to build 9 bars of pressure with ultra-coarse 4:6 method grinds, you are only limited by your ability to press down very quickly or the springs ability to uncoil. Thus, communicating in water debit is not particularly useful for levers, since they truly exert a fixed brew pressure and will supply as much flow as needed in order to maintain their set pressure, meaning the water debit "setting" during the shot pull is unknown. Because a spring will always exert the same amount of pressure, they do not (commonly) have OPV's that limit the maximum brew pressure. In addition, lever machines (dipper levers in particular) are known to have the fastest headspace fill (up to 30-80mL/s depending) because preinfusion is achieved by essentially opening a small hole to a boiler pressurized at 1-1.5 bar. Spring levers in particular will automatically execute a flow profile that preinfuses at high water debit but low pressure (boiler pressure or line pressure), which then leads into a peak pressure which declines slowly as the spring uncoils. 
+
+## Puck Resistance
+
+The pressure at the puck is often treated as a static variable in a commercial setting because it is the pump pressure that is shown. However, it is easy to see with a group pressure gauge that this is not always the case. The resistance in a puck is determined by the grind distribution and the viscosity of the espresso being brewed. It is harder to push honey through a porous medium than it is water, for example. Thus, the pressure generated will depend on grind setting, burr geometry, and the roast profile and origin/variety of the coffee being brewed. At the beginning of a brew when the coffee is at its most soluble, the TDS of the resulting liquid will be very high and output a high pressure. As the viscous solubles of the coffee are depleted, the resulting liquid will often times not flow fast enough to hold the OPV set pressure and the group-pressure reading will begin to fall. Note that on machines which can provide a enough flow, the shot will instead speed up drastically in order to hold pressure. This can often result astringency due to the formation of channels. 
```

Comment:
> which can provide ~~a~~ enough flow

> This can often result astringency due to the formation of channels.

This comment is confusing in isolation. It would be better to explain that flow increasing towards the end of the shot can have two main reasons. TDS reducing or channeling. TDS reducing speeds up flow by reducing the average viscosity while channeling changes the puck structure and therefore changes the effective resistance of the puck.